client/src/api/axios-config.js

```diff
@@ -1,37 +1,35 @@
 import axios from 'axios'
 import Constants from 'expo-constants'
-import { refreshIDToken } from '../utils/auth'
+import { logout } from 'slices/auth.slice'
+import store from '../redux/store'
+import { loadUserIDToken, refreshIDToken } from '../utils/auth'
 
 const API_URL = Constants.manifest.extra.apiURL
 const API_PORT = Constants.manifest.extra.apiDevelopmentPort
 
 // Source: https://stackoverflow.com/questions/47417766/calling-locally-hosted-server-from-expo-app/70964774
 export const BASE_URL = API_URL
   || `http://${Constants.manifest.debuggerHost.split(':').shift()}:${API_PORT}`
+const LOGOUT_MESSAGE = 'Oops, it looks like your login has expired, please try again!'
```

Comment:
change to 'Oops, it looks like your login has expired(in french), please try again!'